test/concatAll.js

```diff
@@ -0,0 +1,35 @@
+import { assert } from 'chai';
+import { NEL, Nil } from 'monet';
+
+import { concatAll } from '../src';
```

Comment:
Use following notation in tests pls. 
```js
import * as RA from '../src';
```

Ref: https://github.com/char0n/ramda-adjunct/blob/2f5173a6699dbf090dee56f389f05bf36dbf66c0/test/isBoolean.js#L4

Reply:
Will do. Why tho?

Reply:
It's a convention we use through out our tests to distinguish between R and RA functions.

Reply:
Done.